src/FutureResult.re

```diff
@@ -1,5 +1,5 @@
 let deprecate = (ft, name, more) =>
-  ft |. Future.tap(_ => Js.Console.warn("FutureResult."++name++" is deprecated." ++ more))
+  ft |. Future.tap(_ => Js.Console.warn("FutureResult."++name++" is deprecated." ++ more));
```

Comment:
whoops.... i didn't mean to commit this. pretty sure i accidentally `bsrefmt`d it